core/src/types/operator/operator.rs

```diff
@@ -927,9 +927,9 @@ impl Operator {
         self.remove_via(stream::iter(paths)).await
     }
 
-    /// remove will given paths.
+    /// remove will remove files via the given paths.
```

Comment:
Is this doc comment written for `remove_via`, if so, the extra empty line should probably be removed

Reply:
I think you may just append `///` at the front of this blank line.

Reply:
Done:)